docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -234,7 +237,7 @@ deprecation ends. For example::
         warnings.warn(
             "foo() is deprecated.",
             category=RemovedInDjangoXXWarning,
-            stacklevel=2,
+            **adjust_stacklevel_for_warning(__file__),
```

Comment:
Seeing this makes me think "should we just have a `deprecate_warn` method that takes most of the `warnings.warn` properties?

Reply:
`deprecate_warn(msg, category=RemovedInDjangoXXWarning)`, and for cases where we don't want to adjust the stack leve.... `deprecate_warn(msg, category=..., adjust_stack=False)`.